docs/api/security/lifecycle/generate_png.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Query: What is this file for? Should it be here?

Reply:
used this script to generate the nice PNG state machine graph.
please suggest the best place for it.